configure.in

```diff
@@ -0,0 +1,6 @@
+AC_INIT(aclocal.m4)
```

Comment:
please don't add back autoconf stuff and empty NEWS files etc.

Reply:
i created a motor project for motor itself

Reply:
and? does that require configure.in ?

Reply:
when creating the motor project it creates them even if it is set to manual makefile mode

Reply:
i removed them as motor seems to be able to build it without configure.in